azurelinuxagent/common/cgroupapi.py

```diff
@@ -386,23 +538,6 @@ def _get_extensions_slice_root_name():
     def _get_extension_slice_name(self, extension_name):
         return "system-{0}-{1}.slice".format(EXTENSIONS_ROOT_CGROUP_NAME, self._get_extension_cgroup_name(extension_name))
 
-    def create_agent_cgroups(self):
```

Comment:
systemd creates the cgroups for the agent so this function was misleading. now, initialize() discovers the cgroups for the agent and starts tracking for them